portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.js

```diff
@@ -1,8 +1,9 @@
+const { isArray } = require('lodash');
```

Comment:
Just because I was curious, I opened a slack thread [here](https://softwire.slack.com/archives/C04N1DJCFAT/p1711363617520929) which @AlexBramhill replied to, so I think we should go with `Array.isArray` :)

Reply:
Oh good spot, thanks